src/1_utils.js

```diff
@@ -750,6 +756,61 @@ utils.removePropertiesFromObject = function(objectToModify, keysToRemove) {
 	}
 };
 
+// v2/event utility functions
+
+var BRANCH_STANDARD_EVENTS = [ 'ADD_TO_CART', 'ADD_TO_WISHLIST', 'VIEW_CART', 'INITIATE_PURCHASE', 'ADD_PAYMENT_INFO', 'PURCHASE', 'SPEND_CREDITS', 'SEARCH', 'VIEW_ITEM', 'VIEW_ITEMS', 'RATE', 'SHARE', 'COMPLETE_REGISTRATION', 'COMPLETE_TUTORIAL', 'ACHIEVE_LEVEL', 'UNLOCK_ACHIEVEMENT' ];
+var BRANCH_STANDARD_EVENT_DATA = [ 'transaction_id', 'revenue', 'currency', 'shipping', 'tax', 'coupon', 'affiliation', 'search_query', 'description' ];
```

Comment:
@dmitrig01  Is this intended to have all the fields listed under event_data in the EO spreadsheet? I see a subset listed here, how do we know that we have the right subset. just trying to get up to speed here.